public/components/player/list/list.html

```diff
@@ -1,14 +1,11 @@
-
 <div id="main"></div>
 
 <script src='../../../node_modules/steal/steal.js' main="@empty">
 	import "bitballs/components/player/list/";
 	import "bitballs/models/fixtures/players";
 	import stache from "can/view/stache/";
-	import route from "can/route/"
 
-	var template = stache('<player-list/>');
+	var template = stache('<player-list is-admin="true"/>');
```

Comment:
I think this should be `{is-admin}="true"`.  It will work the other way, but this will set `isAdmin` to `"true"` not `true`.